docs/library.rst

```diff
@@ -9,15 +9,27 @@ To use the syncronous API you can do the following::
     from vncdotool import api
     client = api.connect('vnchost:display', password=None)
 
-You can then call any of the methods available on 
+You can then call any of the methods available on
 :class:`vncdotool.client.VNCDoToolClient` and they will block until completion.
 For example::
 
     client.captureScreen('screenshot.png')
     client.keyPress('enter')
     client.expectScreen('login_success.png', maxrms=10)
 
-This can be used to automate the starting of an Virtual Machine or other application::
+It is possible to set a per-client timeout in seconds to prevent calls from blocking indefinitely.
+
+::
+
+    client.timeout = 10
+    try:
+        client.captureScreen('screenshot.png')
+    except VNCDoException:
```

Comment:
Seeing this here, I can't help but wonder if we should have a finer grained exceptions, e.g.```class VNCDOTimeout(VNCDoException)```

Reply:
Python 3 provides a `TimeoutError` builtin I'd rather use that than introducing a redundant VNCDoTool based exception.

For Python 2 we could define `TimeoutError` within the module.

Reply:
ok, that sounds reasonable.  I'm not planning a release of vncdotool, but you'd need to make that change before I do one, as Timeout not being part of the VNCDoException hierarchy would be a breaking change.